src/games/strategy/sound/ClipPlayer.java

```diff
@@ -444,22 +523,21 @@ private void parseClipPaths(final String pathName, final boolean subFolder) {
     return availableSounds;
   }
 
+
   private static boolean isSoundFileNamed(final File soundFile) {
     return soundFile.getName().endsWith(".wav") || soundFile.getName().endsWith(".au")
         || soundFile.getName().endsWith(".aiff")
-        || soundFile.getName().endsWith(".midi") || soundFile.getName().endsWith(".mp3");
+        || soundFile.getName().endsWith(".midi");
   }
 
 
   protected static Clip createClip(final URL clipFile) {
     try {
       final AudioInputStream audioInputStream = AudioSystem.getAudioInputStream(clipFile);
-      final AudioFormat audioFormat = audioInputStream.getFormat();
```

Comment:
The mp3 spi change was actually pretty small, just another line to decode the audio format once more. The cleanup to get it so that the change was in exactly one place was the bulk of the work in #201  Going to mp3 from here should be easier at the very least.